city_scrapers/spiders/fortx_Fort_Worth_Isd_Coc.py

```diff
@@ -0,0 +1,116 @@
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class FortxFortWorthIsdCocSpider(CityScrapersSpider):
+    name = "fortx_Fort_Worth_Isd_Coc"
+    agency = "Fort Worth ISD Citizens' Oversight Committee"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.fwisd.org/departments/operations/capital-improvement-program/2021-citizens-oversight-committee-coc"  # noqa
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        location = {
+            "name": "Fort Worth ISD District Service Center",
+            "address": "7060 Camp Bowie Blvd, Fort Worth, TX 76116",
+        }
+
+        # go over the new events
+        for item in response.css(".fsDayContainer"):
```

Comment:
Will do.

Reply:
It looks like that class is still there. This is what I see.

<img width="1440" alt="Screen Shot 2024-12-19 at 2 31 47 PM" src="https://github.com/user-attachments/assets/f3901532-fbd2-49ad-9d28-999fe5cb6d7b" />

Reply:
Thank you for verifying that. The website content matches what you have captured. In the future, I will capture the HTML content to prevent any potential discrepancies between what we see on the page.